d3rlpy/base.py

```diff
@@ -423,7 +442,7 @@ def _evaluate(self, episodes, scorers, logger):
             logger.add_metric(name, test_score)
 
             # store metric locally
-            if name not in self.eval_results_:
+            if name not in self.eval_results_ and test_score is not None:
```

Comment:
Is there any case that `test_score` is `None`?

Reply:
Ok, now is the history. I am using scorers as callbacks after an epoch. I created a fake "scorer" that plots losses and the evaluation of the policy against a reference episode to check not only the loss values but algo the real behavior of the current actor.  So instead of scorers the really usable thing is a kinf of minotor or callback function for each (epoch, step)
Those callbacks (functions) are not required to return a Value.  What do you think ?

Reply:
Sounds nice! I think it's worth having it in `fit` method. Maybe, you can make another PR for the callback, which is like adding `callback` argument to `fit` (and maybe `fit_online`, but this can be later). Then, we can discuss on the callback things there. What does it sound to you?

Reply:
Perfect